interfaces/error/IPageError.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Defines interfaces for page errors.
+ */
+
+export interface IPageError {
+  statusCode: number;
```

Comment:
What do you use here for non-http errors?  Bad dns, invalid url, invalid scheme, etc.

Reply:
My thought here is these codes are for the page being requested, so http status codes seemed like those that would like need to be used here.